releaseNotes/MicrosoftExtensionsConfigurationAzureAppConfiguration.md

```diff
@@ -1,5 +1,16 @@
 ## Microsoft Extensions Azure App Configuration
 
+### 2.0.0-preview-010050001-38 - October 03, 2019
+* Added support for resolving Azure KeyVault references retrieved from configuration store.
+* Fixed the following bugs in the implementation of on-demand refresh mechanism.
```

Comment:
Fixed the following issues when using `ConfigureRefresh' for dynamic configuration
 * Configuration is no longer updated on each refresh invocation if `refreshAll` is `true` for a key that does not exist.